### PR TITLE
Change specid from u32 to text

### PR DESCRIPTION
**Pull request by boxhock**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ console.log(`Submitted with hash ${txHash}`);
 Make sure you add the following additional to `Settings/Developer` section:
 
 ```json
-{"SpecIndex": "u32",
+{"SpecIndex": "Vec<u8>",
```
Comment (j16r) [inline README.md:44]: Should this be `String` instead of `Vec<u8>` which is technically ascii?

Comment (boxhock) [reply README.md:44]: We're not using `String` because it's not supported in WASM, though `Vec<u8>` can represent a string.

Comment (j16r) [reply README.md:44]: Oh, don't they both map onto uint64? I was under the impression wasm only had a handful types: (un)signed int32, 64, float32, 64 (and of course memory regions).

Comment (boxhock) [reply README.md:44]: Regardless I believe it's common to use `Vec<u8>` in Substrate, see: https://github.com/paritytech/substrate/wiki/FAQ

**Pull request merged**